src/material-experimental/mdc-chips/README.md

```diff
@@ -1 +1,109 @@
-This is a placeholder for the MDC-based implementation of chips.
\ No newline at end of file
+This is a prototype of an alternate version of `MatChip` built on top of
+[MDC Web](https://github.com/material-components/material-components-web). This component is experimental and should not be used in production.
+
+## How to use
+Assuming your application is already up and running using Angular Material, you can add this component by following these steps:
+
+1. Install `@angular/material-experimental` and MDC Web:
+
+   ```bash
+   npm i material-components-web @angular/material-experimental
+   ```
+
+2. In your `angular.json`, make sure `node_modules/` is listed as a Sass include path. This is
+   needed for the Sass compiler to be able to find the MDC Web Sass files.
+
+   ```json
+   ...
+   "styles": [
+     "src/styles.scss"
+   ],
+   "stylePreprocessorOptions": {
+     "includePaths": [
+       "node_modules/"
+     ]
+   },
+   ...
+   ```
+
+3. Import the experimental `MatChipsModule` and add it to the module that declares your component:
+
+   ```ts
+   import {MatChipsModule} from '@angular/material-experimental/mdc-chips';
+
+   @NgModule({
+     declarations: [MyComponent],
+     imports: [MatChipsModule],
+   })
+   export class MyModule {}
+   ```
+
+4. Use the chips in your component's template:
+   
+   ```html   
+   <mat-chip-set>
+     <mat-chip> Chip 1 </mat-chip>
+     <mat-chip> Chip 2 </mat-chip>
+   </mat-chip-set>
+   ``` 
+   
+5. Add the theme mixins to your Sass:
+
+   ```scss
+   @import '~@angular/material/theming';
+   @import '~@angular/material-experimental/card-theme';
+
+   $candy-app-primary: mat-palette($mat-indigo);
+   $candy-app-accent:  mat-palette($mat-pink, A200, A100, A400);
+   $candy-app-theme:   mat-light-theme((
+     color: (
+       primary: $candy-app-primary,
+       accent: $candy-app-accent,
+     ),
+   ));
+
+
+   @include mat-mdc-chips-theme($candy-app-theme); 
+   ```
+
+## API differences
+
+The MDC-based chips use a different API than the current chips.
```

Comment:
This could use some clarification, most stuff is the same, just the name of the container changed from `mat-chips` to one of the 3 below, depending on how its being used

Reply:
Re-worded a bit - there's probably a better way to explain how to jump from the legacy implementation to MDC. 

Now we have `mat-chip-set`, `mat-chip-listbox`, and `mat-chip-grid` containers with `mat-chip`, `mat-mat-chip-options`, and `mat-chip-row` chips. 

Previously we just had the `mat-chip-list` container and `mat-chip` chips.